core/database.py

```
@@ -0,0 +1,120 @@
+
+
+def check_db_health() -> str:
+    """
+    Check database health.
+    Returns: 'healthy' | 'degraded' | 'unhealthy'
+    """
+    try:
+        engine = get_engine()
+        with engine.connect() as conn:
+            conn.execute(text("SELECT 1"))
+        return "healthy"
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)